### PR TITLE
fix: Pull arch-specific image tag, instead of multiarch image

### PR DESCRIPTION
**Pull request by untitaker**
ref: INC-604

### CODE REVIEW

**Review by evanh — APPROVED** [review]

**Review by IanWoodard — APPROVED** [review]

**Review by phacops — APPROVED** [review]

**Comment by untitaker** [comment]
we shouldn't merge this unless we are quickly trying to get ahold of a new snuba image because we had to revert a PR there. there's some concerns about drift between devenv and CI if they don't pull the same image tag

**Comment by untitaker** [comment]
gonna try this instead: https://github.com/getsentry/snuba/pull/5365

**Pull request closed without merging**
